### PR TITLE
Pass file size to Iceberg commit task

### PR DESCRIPTION
**Pull request by jackye1995**
Test plan - change covered by existing tests

```
== NO RELEASE NOTE ==
```

Pass file size to commit task so that FileIO does not need to be used during commit phase.

### CODE REVIEW

**Review by ChunxuTang — COMMENTED** [review]
@jackye1995 Thanks for your work! LGTM

File: presto-iceberg/src/main/java/com/facebook/presto/iceberg/CommitTaskData.java [hunk 1]
```diff
@@ -23,16 +23,19 @@
...
 
     @JsonCreator
     public CommitTaskData(
             @JsonProperty("path") String path,
+            @JsonProperty("fileSizeInBytes") long fileSizeInBytes,
             @JsonProperty("metrics") MetricsWrapper metrics,
             @JsonProperty("partitionDataJson") Optional<String> partitionDataJson)
     {
         this.path = requireNonNull(path, "path is null");
+        this.fileSizeInBytes = fileSizeInBytes;
```
Comment (ChunxuTang) [inline presto-iceberg/src/main/java/com/facebook/presto/iceberg/CommitTaskData.java:38]: nit: requiteNonNull?

Comment (jackye1995) [reply presto-iceberg/src/main/java/com/facebook/presto/iceberg/CommitTaskData.java:38]: this is a `long`, cannot be bull

Comment (ChunxuTang) [reply presto-iceberg/src/main/java/com/facebook/presto/iceberg/CommitTaskData.java:38]: Thanks for the clarification!

**Comment by ChunxuTang** [comment]
@jackye1995 Looks like one test fails. Could you force push to rerun the tests?

**Review by zhenxiao — APPROVED** [review]
looks good, thank you @jackye1995

**Pull request merged**